media/server/gstplayer/source/tasks/generic/AttachSource.cpp

```diff
@@ -311,8 +311,8 @@ void AttachSource::switchAudioSource(GstCaps *caps, const std::string &strCaps)
         m_gstWrapper->gstCapsUnref(oldCaps);
     }
     RIALTO_SERVER_LOG_MIL("Switching audio source.");
-    RIALTO_SERVER_LOG_DEBUG("Old caps: %s", oldCapsStr.c_str());
-    RIALTO_SERVER_LOG_DEBUG("New caps: %s", strCaps.c_str());
+    RIALTO_SERVER_LOG_MIL("Old caps: %s", oldCapsStr.c_str());
```

Comment:
I think these should either stay as _DEBUG or combine all 3 log lines into a single _MIL so as not to produce excessive MIL log lines